jobserver/api/releases.py

```diff
@@ -135,6 +135,7 @@ def get_size(rfile):
                 date=rfile.created_at.isoformat(),
                 sha256=rfile.filehash,
                 size=get_size(rfile),
+                is_deleted=not rfile.absolute_path().exists(),
```

Comment:
I'd be inclined to changed this to `is_deleted=file.deleted_at or not rfile.exists()`. The reason being that the failure mode is safer - if we somehow managed to set `deleted_at` and `deleted_by`, but forgot to delete the file, then the users would still be able to see the file. Better to disable the SPA UA on either condition

But a minor corner case, so no worries.